notebooks/Ros2 Pub_Sub_Communication.ipynb

```diff
@@ -0,0 +1,225 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "The rospy package is not found in your $PYTHONPATH. Subscribe and publish are not going to work.\n",
+      "Do you need to activate your ROS environment?\n"
+     ]
+    }
+   ],
+   "source": [
+    "import sys\n",
+    "sys.path.append(\"./../../\")\n",
```

Comment:
Please do not include relative paths. Without any instructions, it is not possible to determine which directory is supposed to be appended.